camunda/camunda-platform-helm
# Add optimize to helm charts

## This PR contains:

* Add a new sub-chart for optimize (most resources were copied from Tasklist/Operate)
* Add default values to the values.yaml file
* Integrate Optimize with Identity (add configuration on both sides)
* Disable optimize if identity authentication is disabled
* Add template tests

## Manual testing:

After installing we see:

![notes](https://user-images.githubusercontent.com/2758593/164460706-e2d556c8-448d-4273-9760-df6c20ab3c8b.png)

Doing the necessary port-forwards we can login

![optimize](https://user-images.githubusercontent.com/2758593/164460435-fa567400-c070-4866-90e6-bcd5215025e0.png)

## Next Steps:

 * We have to solve the issue with the partition count, which is necessary for Optimize #286 
   * Per default optimize will now only import for one partition
 * We will add the values documented to the README #284 
 * We will enhance our e2e/integration test - login to optimize #283 


## Reviewers:

I know it looks like an big PR, but most of the lines are due to tests and golden files, which you can skip. :)

@oleschoenburg @npepinpe whoever has more time please have a look at the PR
@RomanJRW could you please check the configuration to whether this makes sense in your opinion? 


related to #126
